app/views/themes/dc_show/hyrax/base/_representative_media.html.erb

```diff
@@ -0,0 +1,14 @@
+<%#
+  OVERRIDE Hyrax v3.6.0 to check if the work has all attachments that are set to institution.
+  If so then the UV should not show and instead display the thumbnail.
+%>
+
+<% if presenter.representative_id.present? && presenter.representative_presenter.present? %>
+  <% if defined?(viewer) && viewer && (!attachments_with_institution_visibility?(presenter) || current_ability.admin?) %>
```

Comment:
It seems like checking if current user can view the item is better than checking if the user is admin... prob need to verify desired behavior here.